include/miopen/miopen.h

```diff
@@ -1730,6 +1730,14 @@ miopenFindConvolutionForwardAlgorithm(miopenHandle_t handle,
  * The scaling parameter alpha (float) and shift parameter beta (float) are only supported for
  * alpha = 1 and beta = 0.
  *
+ * The forward convolution is designed to accommodate both packed and non-packed tensor strides for
+ * multiple data types and dimensions across various platforms. This flexibility ensures optimal
+ * performance in handling diverse computational scenarios. To configure tensor parameters,
+ * including strides, users can utilize the APIs miopenSetTensorDescriptor() and
+ * miopenGetTensorDescriptor(). These APIs empower developers to seamlessly set and retrieve tensor
+ * information, facilitating a more intuitive and efficient workflow. The tensor strides are
+ * non-packed by default.
+ *
  * If using Group/Depthwise convolution mode, call miopenSetConvolutionGroupCount() before running
  * this.
```

Comment:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/8be2998d7a1210b7bc82d778f7df0d92aa65b57d/include/miopen/miopen.h#L721-L722

This sounds strange: "MIOpen has support for 1, 2, 3, 4, 5 dimensional tensor of layout"

I think it is worth mentioning unpacked tensors here.

Reply:
@junliume @iq136boy is the documentation final?

Reply:
> @junliume @iq136boy is the documentation final?

@Rmalavally @junliume The final version unless there are more comments.

Reply:
Thank you for confirming. Much appreciated. I will now review the content and recommend changes as needed.

Reply:
[resolved]